test/mbr_update_test.rb

```diff
@@ -28,6 +31,21 @@
       )
     end
 
+    context "on s390" do
+      before do
+        allow(Yast::Arch).to receive(:architecture).and_return("s390_64")
```

Comment:
ok, it is fair. Above I use architecture, because I have to mock other cases like `Arch.ppc` in one step, so for generic mock it is needed, for this one, it is not needed, so I modify it.